src/styles/main.scss

```diff
@@ -1 +1,76 @@
-// put code here
+$displaySize: 240px;
+$minuteHeight: $displaySize / 2;
+$minuteWidth: 7px;
+$hourHeight: $minuteHeight / 2;
+$hourWidth: 12px;
+$centerSize: 15px;
+
+* {
+  box-sizing: border-box;
+}
+
+.display {
+  position: relative;
+  height: $displaySize;
+  width: $displaySize;
+  margin: auto;
+  border: 5px dotted #000;
+  border-radius: 50%;
+
+  &__minute {
+    position: absolute;
+    bottom: 50%;
+    left: 49.9%;
+    height: $minuteHeight;
+    width: $minuteWidth;
+    background-color: #f09;
+    animation-name: move;
+    animation-duration: 13s;
+
+    @keyframes move {
+      from {
+        transform: rotate(0deg);
+        transform-origin: bottom;
+      }
+
+      to {
+        transform: rotate(360deg);
+        transform-origin: bottom;
+      }
+    }
+  }
+
+  &__hour {
+    position: absolute;
+    bottom: 50%;
+    left: 48.5%;
+    height: $hourHeight;
+    width: $hourWidth;
+    background-color: #0f7;
+    animation-name: move;
+    animation-duration: 5s;
+
+    @keyframes move {
+      from {
+        transform: rotate(0deg);
+        transform-origin: bottom;
+      }
+
+      to {
+        transform: rotate(360deg);
+        transform-origin: bottom;
+      }
+    }
```

Comment:
Now you are duplicating this animation twice, in this case you can get away with one

Reply:
isn't fixed. 
1. You need to create only one animation and reuse it both to minute hand an hour one.
2. Also you shouldn't use  transform-origin: bottom; inside animation.
3. Move keyframes outside selectors